src/mainviews/detail-view/job-detail.tsx

```diff
@@ -268,6 +270,12 @@ export function JobDetail(props: IJobDetailProps): JSX.Element {
   return (
     <>
       {ButtonBar}
+      {props.model.statusMessage && (
+        <Alert severity="error">
+          <AlertTitle>Error</AlertTitle>
```

Comment:
```suggestion
          <AlertTitle>{trans.__('Error')}</AlertTitle>
```

Reply:
For now, remove `AlertTitle ` altogether